eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -204,40 +204,24 @@ jobs:
               Write-Host "##vso[task.setvariable variable=AZURE_SDK_VERSION]$version"
             displayName: Create package info JSON file
 
-          # Building from the root of the repo at this point because of the
-          # nature of the CMake build system. The SoureDirectory should probably
-          # change.
-          - task: PowerShell@2
-            inputs:
-              targetType: filePath
-              filePath: eng/scripts/New-ReleaseArtifact.ps1
-              arguments: >-
-                -SourceDirectory ./
-                -PackageSpecPath $(Build.ArtifactStagingDirectory)/packages/${{ artifact.Name }}/package-info.json
-                -Destination $(Build.ArtifactStagingDirectory)/packages/${{ artifact.Name }}/vcpkg
-                -Workspace $(Agent.TempDirectory)/vcpkg-${{ artifact.Name }}
-              pwsh: true
-            displayName: Compress publish artifact and stage for publishing
-
           - task: Powershell@2
             inputs:
               targetType: inline
               script: |
                 # If there is no vcpkg directory skip staging the vcpkg port artifacts
-                if (!(Test-Path sdk/${{ parameters.ServiceDirectory }}/${{ artifact.Name }}/vcpkg/)) {
+                if (!(Test-Path build/vcpkg/ports/${{ artifact.VcpkgPortName }})) {
                   exit
                 }
 
                 New-Item `
                   -ItemType Directory `
-                  $(Build.ArtifactStagingDirectory)/packages/${{ artifact.Name }}/vcpkg/port -Force
+                  $(Build.ArtifactStagingDirectory)/packages/${{ artifact.Name }}/vcpkg -Force
 
-                Copy-Item `
-                  sdk/${{ parameters.ServiceDirectory }}/${{ artifact.Name }}/vcpkg/* `
-                  $(Build.ArtifactStagingDirectory)/packages/${{ artifact.Name }}/vcpkg/port `
-                  -Exclude README.md
```

Comment:
Why did we have to exclude `README` before and why can we get rid of it now?

Reply:
This is old code. The cmake process selects the files to move to the port folder and can exclude README.md on its own. 

The README.md is supposed to indicate that there are no directly consumable vcpkg assets in the `vcpkg` folder [example](https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/template/azure-template/vcpkg/README.md) 

.. We should have that README.md file in all of the vcpkg folders